README.md

```diff
@@ -26,48 +25,40 @@ jupyterlab-requiremnts>=0.4.5
 
 ### Operate First Open Environment
 
-[Operate First][2] Open Infrastructure environment has been selected to run this tutorial. It fullfills all the requirement stated above. If you are interested in using it, just get in touch with Operate First team, it's an open source initiative.
+[Operate First][2] is an open infrastructure environment started at Red Hat's Office of the CTO. It has been selected to run this tutorial since it is an open source initiative that fulfills all the requirements stated above. Anyone with a Google account can log in and start developing. To learn more about Operate First, visit the [website](https://www.operate-first.cloud/) or [GitHub community](https://github.com/operate-first).
 
-You can find some notes also regarding other environments in the different sections of the tutorial.
 
 ## Tools
 
 In this tutorial the following technologies are going to be used:
 
-- [JupyterHub][4], to launch images with Jupyter tooling.
-- [Elyra][5], which is a set of AI-centric extensions to JupyterLab Notebooks (e.g. interface with Kubeflow Pipeline, Git, Python scripts).
-- [Project Thoth][6] Extension for Dependency Management on JupyterLab. If you want to know more, have a look at this [repo]((https://github.com/thoth-station/jupyterlab-requirements)).
-- [Kubeflow Pipelines][9], to allow end to end experiment using pipelines.
+- [JupyterHub][4] to launch images with Jupyter tooling
+- [Elyra][5] supplies a set of extensions to JupyterLab notebooks to support AI projects
+- [Project Thoth][6] extension for dependency management on JupyterLab
+- [Kubeflow Pipelines][9] for end to end experiments using pipelines
 
-## GitOps, reproducibility, portability and traceability with AI support
```

Comment:
I don think we want to remove "with AI Support" as that is a reference to Thoth's abilities later in this section.